src/libraries/System.Net.Requests/src/System/Net/RequestStream.cs

```diff
@@ -105,6 +105,11 @@ public override Task WriteAsync(byte[] buffer, int offset, int count, Cancellati
             return _buffer.WriteAsync(buffer, offset, count, cancellationToken);
         }
 
+        public override ValueTask WriteAsync(ReadOnlyMemory<byte> buffer, CancellationToken cancellationToken = default)
+        {
+            return _buffer.WriteAsync(buffer, cancellationToken);
+        }
+
```

Comment:
These System.Net.Requests changes aren't related to "DeflateStream.CopyToStream", are they?

Reply:
No, I updated PR title to clarify that.